cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1415,7 +1414,7 @@ def wait_for_input(self,duration=0):
 			rospy.logerr("Wait with duration not implemented yet") # \todo TODO: implement waiting with duration
 
 		rospy.loginfo("Wait for user input...")
-		retVal = raw_input()
+		retVal = input()
```

Comment:
according to https://github.com/mojin-robotics/mojin_manipulation/pull/82#discussion_r371215058 this needs to be reverted.